src/digitalmarketplace/components/alert/_alert.scss

```diff
@@ -1,5 +1,5 @@
-@import "govuk/tools/all";
-@import "govuk/helpers/all";
+@import "node_modules/govuk-frontend/tools/all";
```

Comment:
As we're copying govuk-frontend to `src` and adding `src` to the Sass `includePaths` we shouldn't need to change these paths right?

Reply:
Ah, correct - this is a hangover from before I knew about prefixLoader for Briefs